crates/bevy_ecs/src/system/query.rs

```diff
@@ -488,6 +488,33 @@ where
         }
     }
 
+    /// Gets the result of a single-result query.
+    ///
+    /// Assumes this query has only one result and panics if there are no or multiple results.
+    /// If you want to handle the error case yourself you can use the [`Self::try_single`] variant.
+    ///
+    /// # Example
+    ///
+    /// ```
+    ///  # use bevy_ecs::system::{Query, QuerySingleError};
```

Comment:
Why do we import `QuerySingleError`?

Reply:
Good catch, I simply pasted the original comment and forgot to remove that, fixed.